src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -582,6 +585,22 @@ private function createEncoder($config, ContainerBuilder $container)
             ];
         }
 
+        // Argon2id encoder
+        if ('argon2id' === $config['algorithm']) {
+            if (!Argon2idPasswordEncoder::isSupported()) {
+                throw new InvalidConfigurationException('Argon2i algorithm is not supported. Install the libsodium extension or use BCrypt instead.');
+            }
+
+            return [
+                'class' => 'Symfony\Component\Security\Core\Encoder\Argon2idPasswordEncoder',
```

Comment:
Can use `::class` as Symfony 3.4 is PHP 5.5+

Reply:
fixed thanks